ReactNativeClient/lib/import-enex-md-gen.js

```diff
@@ -395,7 +395,15 @@ function isSpanStyleBold(attributes) {
 		//console.debug('font-family regex matched');
 		return true;
 	} else {
-		//console.debug('Found unsupported style(s) in span tag: %s', style);
+		return false;
+	}
+}
+
+function isSpanStyleItalic(attributes) {
+	let style = attributes.style;
+	if (style.includes('font-style: italic;')) {
```

Comment:
That seems a bit fragile - what if it is set as `font-style:italic ;`? Maybe spaces should be removed and the string made lowercase before testing it?